estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name':'Real Estate',
+    'version':'0.0.0',
```

Comment:
```suggestion
    'version':'1.0',
```
please add correct version for module.

Reply:
Suggested changes are done.